pytorch/training/docker/1.7/py3/cu110/Dockerfile.gpu

```diff
@@ -244,6 +245,10 @@ RUN SMDATAPARALLEL_PT=1 pip install --no-cache-dir ${SMDATAPARALLEL_BINARY}
 
 ENV LD_LIBRARY_PATH="/opt/conda/lib/python3.6/site-packages/smdistributed/dataparallel/lib:$LD_LIBRARY_PATH"}
 
+# Install PT S3 plugin
+RUN pip install --no-cache-dir -U ${PT_S3_WHL_GPU}
+RUN mkdir -p /etc/pki/tls/certs && cp /etc/ssl/certs/ca-certificates.crt /etc/pki/tls/certs/ca-bundle.crt
```

Comment:
nit: Please break this into 2 different lines like other multi-command `RUN` statements.